lib/MusicBrainz/Server/EditSearch/Predicate/VoteCount.pm

```diff
@@ -0,0 +1,56 @@
+package MusicBrainz::Server::EditSearch::Predicate::VoteCount;
+use Moose;
+use namespace::autoclean;
+
+use MusicBrainz::Server::Constants qw( :vote );
+use MusicBrainz::Server::Types qw( VoteOption );
+use MusicBrainz::Server::Validation qw( is_integer );
+
+extends 'MusicBrainz::Server::EditSearch::Predicate::ID';
+
+around operator_cardinality_map => sub {
+    my ($orig, $self) = splice(@_, 0, 2);
+    my %orig_map = $self->$orig;
+    return map { $_ => 1 + $orig_map{$_} } keys %orig_map;
```

Comment:
What matters though is the order arguments are filed into the SQL query. Instead of making that `args.2`, I moved `args.0` to `vote` and dropped this method modifier in https://github.com/metabrainz/musicbrainz-server/pull/497/commits/3afe77594eb5359c98feb72bc8e329f66cf3a095.